ui/imports/shared/popups/send/panels/HoldingItemSelector.qml

```diff
@@ -43,13 +43,17 @@ Item {
     implicitHeight: comboBox.implicitHeight
 
     onSelectedItemChanged: {
-        d.iconSource = itemIconSourceFn(selectedItem) ?? defaultIconSource
-        d.text = itemTextFn(selectedItem) ?? placeholderText
+        let iconSource = itemIconSourceFn(selectedItem)
+        d.iconSource =  !selectedItem ? "" : !!iconSource ? iconSource : defaultIconSource
```

Comment:
for a string` ?? ` doesn't work it only checks undefined or null.

Reply:
`property var selectedItem`